andela/ah-frontend-thanos
# #161348745 Setup Redux

#### What does this PR do?
This PR sets up Redux with ReactJS

#### Description of Task to be completed?
- Install Redux and React-Redux
- Connect NavBar component to redux for a demonstration
- Add reducers, actions, containers, and components

#### How should this be manually tested?
- Install dependencies using `npm install`
- Running the app with `npm start` should display a welcome message that is collected from a `prop` using the Redux connection

#### Any background context you want to provide?
- The message displayed is *just a demonstration* of one way to use the React-Redux connection.

#### What are the relevant pivotal tracker stories?
[#161348745](https://www.pivotaltracker.com/story/show/161348745)
